components/homme/cmake/Kokkos.cmake

```diff
@@ -1,12 +1,15 @@
 set (KOKKOS_INSTALLATION_NEEDED FALSE CACHE LOGICAL "")
-if (NOT DEFINED KOKKOS_PATH)
+if (NOT DEFINED E3SM_KOKKOS_PATH)
   # Build kokkos submodule if user did not specify KOKKOS_PATH.
   set (KOKKOS_INSTALLATION_NEEDED TRUE)
   set (KOKKOS_SRC ${CMAKE_SOURCE_DIR}/../../externals/kokkos)
   set (KOKKOS_INSTALL_DIR ${CMAKE_BINARY_DIR}/kokkos/install)
   SET (KOKKOS_INCLUDE_DIR ${KOKKOS_SRC}/core/src ${KOKKOS_INSTALL_DIR})
   SET (KOKKOS_LIBRARY_DIR ${KOKKOS_INSTALL_DIR})
+  message ("Kokkos will be built internally to E3SM, in ${KOKKOS_INSTALL_DIR}")
+  # SET (E3SM_KOKKOS_PATH ${KOKKOS_INCLUDE_DIR} CACHE PATH "Kokkos installation directory for E3SM")
```

Comment:
Why is there this commented-out, but new, line?

Reply:
Argh. I added it, then commented (it gave the wrong result), then forgot to remove it. I fixed it now.